apps/landing-page/content/create-whatsapp-chatbot.mdx

```diff
@@ -0,0 +1,301 @@
+   />
+
+6. Enable the WhatsApp product within your app.
+7. Great! Your WhatsApp app is now created
+
+#### Connecting Your Typebot Chatbot to WhatsApp
+
+Let’s connect it to Typebot:
+
+1. Go to your [System users page](https://business.facebook.com/settings/system-users)
+2. Create a new user by clicking on Add
+3. Fill it with any name and give it the Admin role
+4. Click on Add assets. Under Apps, look for your previously created app, select it and check Manage app
+   <Image
+     src="/images/blog/create-whatsapp-chatbot/whatsapp-system-user-assets.png"
+     alt="Meta interface to assign assets to users"
+     width={800}
+     height={479}
+   />
+5. Generate a token and put it in Typebot
+6. Go to your WhatsApp Developer Console and add your phone number
+7. Then, Typebot gives you a webhook URL and a Verify Token. Copy these and paste them into the WhatsApp Developer Console.
+
+You should be fine by now, you can test your chatbot now!
+
+<Image
+  src="/images/blog/create-whatsapp-chatbot/result-example.jpg"
+  alt="WhatsApp result of the chatbot flow example"
+  width={565}
+  height={703}
+/>
+
+## Testing and Deploying Your WhatsApp Chatbot
+
+After pouring your creativity and effort into designing an engaging WhatsApp chatbot, it's time to test it and prepare for deployment. This crucial phase ensures that your chatbot functions seamlessly and provides a polished experience.
+
+### Testing Your Chatbot on WhatsApp
+
+Before launching your chatbot, thoroughly test it in the WhatsApp environment. Typebot's "Test" feature allows you to simulate your chatbot's behavior and interactions. This gives you a chance to identify and address any potential issues.
+
+<Image
+  src="/images/blog/create-whatsapp-chatbot/typebot-share-button.png"
+  alt="Typebot share button"
+  width={289}
+  height={49}
+/>
+
+To access the WhatsApp preview:
+
+1. Click on the "Test" button in the Typebot editor.
+2. Change the runtime to "WhatsApp."
+
+   <Image
+     src="/images/blog/create-whatsapp-chatbot/typebot-preview-whatsapp.jpg"
+     alt="Typebot preview chatbot on WhatsApp"
+     width={498}
+     height={335}
+   />
+
+This will enable you to experience your chatbot as if you were a user interacting with it on WhatsApp.
+
+During testing, pay close attention to:
+
+- **User Experience**: Ensure the conversation flow is natural, intuitive, and free from confusing transitions.
+- **Media Rendering**: Verify that images, videos, and other media assets display correctly.
+- **Button Functionality**: Test each button to ensure it triggers the appropriate response or action.
+- **Input Field Validation**: Check that input fields function correctly, validating user input and triggering appropriate conditional logic.
+- **Edge Cases**: Explore potential edge cases or unexpected user inputs to ensure graceful handling and appropriate responses.
+
+Document any issues or areas for improvement. This will help streamline debugging and optimization.
+
```

Comment:
I think the entire `### Integrating Typebot with WhatsApp` section should be positioned here, after the testing phase. It makes more sense to me 🙏